module.f.ts

```diff
@@ -0,0 +1,42 @@
+import * as tokenizer from './djs/tokenizer/module.f.ts'
```

Comment:
Rename the file to `module.ts`. We use `.f.ts` for pure FunctionalScript without I/O.